Create OSTE_Meta_Scanner.md  "Creating Page that describe OSTE meta scanner project"

OSTEsayed opened the pull request.

kingthorin commented:
Is this an OWASP project?

kingthorin commented on pages/OSTE_Meta_Scanner:
```diff
@@ -0,0 +1,58 @@
+- [Introduction](#introduction)
+- [Features](#features)
+- [License](#license)
+- [References](#References)
```
This list doesn't match the content

OSTEsayed replied:
Thanks for noticing, I fixed the list.

OSTEsayed commented:
> Is this an OWASP project?
No, it's my master's graduation project.

kingthorin commented:
We don't have "product" pages for any other tool (commercial, school project, open source, or otherwise). Is there some need for this? There are a few pages that list/link tools can't this just be added/referenced like others where applicable?

OSTEsayed commented:
> We don't have "product" pages for any other tool (commercial, school project, open source, or otherwise). Is there some need for this? There are a few pages that list/link tools can't this just be added/referenced like others where applicable?

I thought it would be better if I described it, but after you mentioned it, do you think it's a good idea to add it to /_data/tools.json instead?

kingthorin commented:
I think that would make more sense and follow the existing pattern more.

OSTEsayed commented:
> I think that would make more sense and follow the existing pattern more.

Done, I have deleted the new page and updated the tool.json file. Thank you for your guidance and time I really appreciate it.

kingthorin commented on _data/tools.json:
```diff
@@ -1,4 +1,13 @@
+      "owner": "OSTEsayed",
+      "license": "Open Source",
+      "platforms": "Linux",
+      "note": "OSTE meta scanner is a comprehensive web vulnerability scanner that combines multiple DAST scanners, including Nikto Scanner, OWASP ZAP, Nuclei, SkipFish, and Wapiti.",
```
```suggestion
      "note": "OSTE meta scanner is a comprehensive web vulnerability scanner that combines multiple DAST scanners, including Nikto Scanner, ZAP, Nuclei, SkipFish, and Wapiti.",
```
